samples/aspnetcore/blazor/Validation/Pages/_Host.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@page "/"
+@namespace Validation.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Validation</title>
+    <base href="~/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="css/site.css" rel="stylesheet" />
+</head>
+<body>
+    <app>
+        @* Remove the following line of code to disable prerendering *@
+        @(await Html.RenderComponentAsync<App>())
```

Comment:
Is it intentional to build this on preview 8? L17-18 would need to change when we update to preview 9. Could it just be done on preview 9 already?